woocommerce/woocommerce
# Add product card styling.

### All Submissions:






### Changes proposed in this Pull Request:
- Update product cards on the WooCommerce > Extensions page in WP Admin to match current designs.


### How to test the changes in this Pull Request:

1. Check out this branch on local environment.
2. Navigate to one of marketplace category pages, i.e. `wp-admin/admin.php?page=wc-addons&section=payment-gateways`  (nav menu isn't implemented at the moment
3. Check product card and make sure it looks good on all screen sizes

### Other information:

* [ ] Have you written new tests for your changes, as applicable? - e2e tests are currently failing because the WooCommerce.com Subscriptions screen is now in a different place, these changes are addressed in #30380.



### Screenshots

![](https://d.pr/i/zpIwTT+)

### FOR PR REVIEWER ONLY:
